backends/arm/test/models/test_mobilenet_v2_arm.py

```diff
@@ -30,37 +31,39 @@ class TestMobileNetV2(unittest.TestCase):
         "executorch_exir_dialects_edge__ops_aten_convolution_default",
     }
 
-    @unittest.skip("This test is not supported yet")
+    operators_after_quantization = all_operators - {
```

Comment:
Unrelated changes? Please move them to a different PR?

Reply:
i would suggest put it here since for the following reasons: 
- to avoid unnecessary logistical overheads
- this patch's main goal is to support MNV2, we need this change for MNV2 unittest case